feat: add hasura's blog post (#127)
cloudnative-pg/cloudnative-pg.github.io: content/authors/leonardoce/index.md

---
name:  Leonardo Cecchi
avatar: leonardoce-gh.jpeg
github: leonardoce
---

I’ve been in computer science from the beginning, and programming has always
been my obsession. My passion began when I was 6 when my father taught me the
rudiments of C programming.

I’ve been working in 2ndQuadrant since 2016 as a developer and a PostgreSQL
consultant. PostgreSQL has always been my choice when working with a DBMS. I’ve
been using it since 1998. 

> EDB acquired 2ndQuadrant in 2020 and CloudNative-PG, the operator me and my team
> has been working on since 2019, was finally released as OSS in 2022.

Reviewer: ```suggestion
EDB acquired 2ndQuadrant in 2020, and CloudNativePG, the operator I
have been working on with my team since 2019, was finally released as OSS in 2022.
```

I’m working now as a maintainer for the CloudNative-PG project, as a developer

> for EDB PostgreSQL Distributed, and as an architect for BigAnimal, the DBaaS
> offering of EDB.

Reviewer: ```suggestion
for EDB PostgreSQL Distributed, and as an architect for BigAnimal, EDB's
DBaaS  offering.
```
